src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -278,7 +278,13 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        check(find(by.xpath("//input[@name='parameterized']")));
```

Comment:
If I am not mistaken it can be replaced by `control("/properties/hudson-model-ParametersDefinitionProperty/specified", "//input[@name='parameterized']").check()`